metrics/config.go

```diff
@@ -147,15 +159,42 @@ func NewStackdriverClientConfigFromMap(config map[string]string) *StackdriverCli
 // Record applies the `ros` Options to `ms` and then records the resulting
 // measurements in the metricsConfig's designated backend.
 func (mc *metricsConfig) Record(ctx context.Context, ms stats.Measurement, ros ...stats.Options) error {
+	var err error
```

Comment:
this can be folded into the `if`.

Reply:
Thanks, I'd previously had the `if` on one line with the assignment, but realized I needed to avoid aliasing `ros`.